docs/how-to/reject-client-connections.md

```diff
@@ -0,0 +1,56 @@
+# Rejecting Client Connections
+
+SpacetimeDB provides a way to disconnect a client during a client connection attempt.
+
+:::server-rust
+In Rust, if we returned and error (or a panic) during the `client_connected` reducer, the client will be disconnected.
+
+Here is a simple example where the server module throws an error for all incoming client connections.
+```rust
+#[reducer(client_connected)]
+pub fn client_connected(_ctx: &ReducerContext) -> Result<(), String> {
+    let client_is_rejected = true;
+    if client_is_rejected {
+        Err("The client connection was rejected. With our current code logic, all clients will be rejected.".to_string())
+    } else {
+        Ok(())
+    }
+}
+```
+
+Client behavior can vary by client type. For example:
+* **C# clients**: Client disconnection behavior is currently undefined and will generate an error reading:
+  `Unable to send subscribe message: WS sender loop has dropped its recv channel: TrySendError { kind: Disconnected }`
```

Comment:
This seems to reference a Rust error. I would be surprised if this happened on C# clients.